packages/ppx/test/css-support/color-module.t/run.t

```diff
@@ -146,5 +146,5 @@ If this test fail means that the module is not in sync with the ppx
   CSS.unsafe({js|columnRuleColor|js}, {js|#000000FF|js});
   CSS.unsafe({js|columnRuleColor|js}, {js|rebeccapurple|js});
   
-  CSS.color(`rgba((0, 0, 0, `num(1.))));
+  CSS.color(`rgba((0, 0, 0, `calc(`num(1.)))));
```

Comment:
I don't think this is correct, calc shouldn't leak here

Reply:
So calc(1) should evaluate to 1?

Reply:
Nevermind, I read the diff wrong. I thought calc wasn't on the value.